website_event_require_login/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Website Event Require Login",
+    "version": "14.0.1.0.0",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

Could you remove this please? :)